internal/cmd/dev/cmd.go

```diff
@@ -0,0 +1,26 @@
+package dev
+
+import (
+	"strings"
+
+	"github.com/gobuffalo/events"
+	"github.com/spf13/cobra"
+)
+
+var (
+	debug bool
```

Comment:
I know the default value is false like with `yesToAll` but having it explicitly would better document the behaviour imho